src/Makefile.qt.include

```diff
@@ -38,6 +38,7 @@ QT_MOC_CPP = \
   qt/moc_bitcoinamountfield.cpp \
   qt/moc_bitcoin.cpp \
   qt/moc_bitcoingui.cpp \
+  qt/moc_bitcoinmobilegui.cpp \
```

Comment:
Avoid "mobile" for now? I'd prefer "touch".

Reply:
Agreed that it shouldn't be "mobile". But "touch" implies touchscreens when it's perfectly usable with a mouse & keyboard combo. perhaps "qml"? "modern"? "nextgen"? :smile:

Reply:
🤷